<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(55, 55, 55);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);</string>
   </property>
   <widget class="QPushButton" name="colorButton1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>630</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>630</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>680</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>680</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="changeColorButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Change Color</string>
    </property>
   </widget>
   <widget class="QComboBox" name="spriteSizeComboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gridToggleButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Toggle Grid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playAnimationButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>210</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Start or Stop Animation</string>
    </property>
   </widget>
   <widget class="QSlider" name="fpsSlider">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>240</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fpsLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>240</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>fps</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentFpsLabel">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>240</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addNewFrame_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Add New Frames</string>
    </property>
   </widget>
   <widget class="QComboBox" name="framescomboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFrame_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Remove frame</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>60</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="penSizeBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>42</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="penSizeLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pen Size</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="penButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Pen</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="eraserButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Eraser</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="selectFrameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Frame</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="animationSizeCheckBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Animate Real Sprite Size</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="exportSizeCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Export Real Size Sprite Image </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
